Source/ALS/Private/RootMotionSources/AlsRootMotionSource_Mantling.cpp

```diff
@@ -47,6 +47,16 @@ void FAlsRootMotionSource_Mantling::PrepareRootMotion(const float SimulationDelt
 
 	const auto* Montage{MantlingSettings->Montage.Get()};
 	const auto MontageTime{MontageStartTime + GetTime() * Montage->RateScale};
+		
+	const auto TargetAnimationLocation{UAlsUtility::ExtractLastRootTransformFromMontage(Montage).GetLocation()};
+	const auto CurrentAnimationLocation{UAlsUtility::ExtractRootTransformFromMontage(Montage, MontageTime).GetLocation()};
+
+	if (FMath::IsNearlyZero(TargetAnimationLocation.Z))
+	{
+		UE_LOG(LogRootMotion, Warning, TEXT("FAlsRootMotionSource_Mantling::PrepareRootMotion fail : %s"), *Montage->GetFName().ToString());
+		RootMotionParams.Clear();
+		return;
+	}
```

Comment:
I decided to use ExtractLastRootTransformFromMontage instead of ExtractRootTransformFromMontage, and I changed the position in the source for early return.